docs/_guides/codewind-vs-code-quick-guide.md

```diff
@@ -61,28 +61,28 @@ If your organization uses customized application stacks and gives you a URL that
 ### Creating an Appsody project
 Throughout the application lifestyle, Appsody helps you develop containerized applications and maximize containers curated for your usage. If you want more context about Appsody, see the [Appsody welcome page](https://appsody.dev/docs).
 
-1. Under the Explorer pane, select `Codewind`.
-2. Expand `Codewind` by clicking the drop-down arrow.
-3. Hover over the `Projects` entry underneath Codewind in the Explorer pane, and press the `+` icon to create a project.
+1. Under the Explorer pane, select **Codewind**.
+2. Expand **Codewind** by clicking the drop-down arrow.
+3. Hover over the **Projects** entry underneath Codewind in the Explorer pane, and press the `+` icon to create a project.
```

Comment:
- Remove "underneath Codewind" for accessibility.
- Comma not needed after "pane."
- The + should be bold, not code phrase format.